assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
A different implementation might have this restriction, also it's turned
off by default (if value equals 0)

2016-04-07 16:09 GMT+02:00 Matthew Buckett <notifications@github.com>:

> In
> assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java
> <https://github.com/ox-it/wl-assignment/pull/31#discussion_r58877047>:
>
> > @@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
> >  			try {
> >  				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
> >  				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
> > -					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
> > +					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
>
> Don't you only want to display this alert when they actually have selected
> to use Turnitin?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/wl-assignment/pull/31/files/6c849e75a7fc9db451dc0fe971510d1afc09b3d4#r58877047>
>